src/goe/offload/offload_constants.py

```diff
@@ -324,13 +317,6 @@
         See GOE-1182 for example of why this is needed
 """
 CAPABILITY_SQL_MICROSECOND_PREDICATE = "sql_pred_microseconds"
-""" CAPABILITY_SYNTHETIC_BUCKETING
-        Does the backend require GOE's standard partitioning scheme for sharding offloaded data
-        (known as offload buckets), applicable only to Hadoop-based systems and used to enable
-        parallel query access to offloaded data. If the backend provides native parallel-read
-        capabilities this is not required.
-"""
-CAPABILITY_SYNTHETIC_BUCKETING = "synthetic_bucketing"
 """ CAPABILITY_SYNTHETIC_PARTITIONING
         Does the backend support GOE's synthetic partition keys (e.g. GOE_PART_M_TIME_ID or
         GOE_PART_U0_SOURCE_CODE) which are used internally by Smart Connector to generate
```

Comment:
If we haven't got one already, we could do with an issue for removal of Smart Connector references.

Reply:
We don't have one but there weren't many references so I've removed them on this branch.